src/Shared/Microsoft.AspNetCore.Razor.Test.Common/Language/IntegrationTests/IntegrationTestBase.cs

```diff
@@ -22,13 +22,8 @@
 
 namespace Microsoft.AspNetCore.Razor.Language.IntegrationTests;
 
-[InitializeTestFile]
-// These tests must be run serially due to the test specific FileName static var.
-[Collection("IntegrationTestSerialRuns")]
 public abstract class IntegrationTestBase
 {
-    private static readonly AsyncLocal<string> _fileName = new AsyncLocal<string>();
-
     // UTF-8 with BOM
```

Comment:
Note that because we don't share a common base type, the tooling version of this type won't see this changes. That's fine with me for now, but we should definitely look into sharing more code in the future. The main problem being that the tooling layer version of this type adds things like JTF and other helpers that are not useful at the compiler layer.